doc/security.md

```diff
@@ -236,11 +236,11 @@ instance is using, or can be added to individual instances, as shown below.
 
 The following security features are available for `bridged` NICs:
 
-Key                      | Type      | Default           | Required  | Description
-:--                      | :--       | :--               | :--       | :--
-security.mac\_filtering  | boolean   | false             | no        | Prevent the instance from spoofing another's MAC address
-security.ipv4\_filtering | boolean   | false             | no        | Prevent the instance from spoofing another's IPv4 address (enables mac\_filtering)
-security.ipv6\_filtering | boolean   | false             | no        | Prevent the instance from spoofing another's IPv6 address (enables mac\_filtering)
+Key                     | Type      | Default           | Required  | Description
+:--                     | :--       | :--               | :--       | :--
+security.mac_filtering  | boolean   | false             | no        | Prevent the instance from spoofing another's MAC address
+security.ipv4_filtering | boolean   | false             | no        | Prevent the instance from spoofing another's IPv4 address (enables mac\_filtering)
+security.ipv6_filtering | boolean   | false             | no        | Prevent the instance from spoofing another's IPv6 address (enables mac\_filtering)
```

Comment:
Those need to be escaped as they aren't inside a code block.

Not all markdown parsers are picky about this, but some definitely are :)

Reply:
Ah, right only need to remove them from the ones outside the table. Done - thanks!